packages/proto-loader/src/index.ts

```diff
@@ -18,7 +18,7 @@
 import * as Protobuf from 'protobufjs';
 import * as fs from 'fs';
 import * as path from 'path';
-import * as _ from 'lodash';
+import camelCase = require('lodash.camelcase');
```

Comment:
Yeah, it is a little weird. It's like this because the DefinitelyTyped package uses the `export = ` syntax instead of es modules syntax:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/lodash.camelcase/index.d.ts

The behavior is described here under "export = and import = require()":
https://www.typescriptlang.org/docs/handbook/modules.html